### PR TITLE
Org rename: github.com/sqlcollaborative -> github.com/dataplat

### PR DESCRIPTION
**Pull request by potatoqualitee**
To better align with our updated org name

### CODE REVIEW

**Comment by potatoqualitee** [comment]
hey @niphlod could you give this a look over? once it's merged, i'll run my release tool which triggers your doc maker, then I'll submit a PR for any /sqlcollaborative references that are still left in the docs repo /cc @wsmelton

File: bin/dbatools-index.json [hunk 1]
```diff
@@ -47162,7 +47162,7 @@
                        ],
                        [
                            "InputObject",
-                           "Enables piped input of Microsoft.SqlServer.Management.Smo.Database, Microsoft.SqlServer.Management.Smo.Server, and Sqlcollaborative.Dbatools.Parameter.DbaInstanceParameter objects.",
+                           "Enables piped input of Microsoft.SqlServer.Management.Smo.Database, Microsoft.SqlServer.Management.Smo.Server, and dataplat.Dbatools.Parameter.DbaInstanceParameter objects.",
```
Comment (niphlod) [inline bin/dbatools-index.json:47165]: hum...

File: internal/functions/Get-DbaHelp.ps1 [hunk 2]
```diff
@@ -199,7 +199,7 @@ function Get-DbaHelp {
             }
             $null = $rtn.Add('')
             $null = $rtn.Add("`n" + '&nbsp;' + "`n")
-            $null = $rtn.Add('Want to see the source code for this command? Check out [' + $doc_to_render.CommandName + '](https://github.com/sqlcollaborative/dbatools/blob/master/functions/' + $doc_to_render.CommandName + '.ps1) on GitHub.')
+            $null = $rtn.Add('Want to see the source code for this command? Check out [' + $doc_to_render.CommandName + '](https://github.com/dataplat/dbatools/blob/master/functions/' + $doc_to_render.CommandName + '.ps1) on GitHub.')
             $null = $rtn.Add("<br>")
             $null = $rtn.Add('Want to see the Bill Of Health for this command? Check out [' + $doc_to_render.CommandName + '](https://sqlcollaborative.github.io/boh#' + $doc_to_render.CommandName + ').')
```
Comment (niphlod) [inline internal/functions/Get-DbaHelp.ps1:204]: this could need additional love, and also the assets over there, but can be tackled when switched over

**Review by niphlod — COMMENTED** [review]
see the two notes ... some adjustments may be needed on the other repos, if the url changes too some of the links could/should be adjusted

File: bin/dbatools-index.json (hunk 1 above)
Comment (wsmelton) [reply bin/dbatools-index.json:47165]: To update the class name we'd have to update it in the library everywhere.

Comment (potatoqualitee) [reply bin/dbatools-index.json:47165]: Thanks for the keen eye!

Comment (potatoqualitee) [reply bin/dbatools-index.json:47165]: closing this PR and starting over, brb

**Comment by potatoqualitee** [comment]
K clean replace found here: https://github.com/dataplat/dbatools/pull/8074

**Pull request closed without merging**